Cabal/src/Distribution/SPDX/LicenseId.hs

```diff
@@ -497,14 +497,15 @@ licenseIdMigrationMessage = go where
     go l | gnuVariant l    = "SPDX license list 3.0 deprecated suffixless variants of GNU family of licenses. Use " ++ l ++ "-only or " ++ l ++ "-or-later."
     go "BSD3"              = "Do you mean BSD-3-Clause?"
     go "BSD2"              = "Do you mean BSD-2-Clause?"
+    go "LGPL-2"            = "Do you mean LGPL-2.0-only?"
```

Comment:
```
LGPL-2.1-only or LGPL-3.0-only or -or-later variants
```

for old formats cabal says

```
Warning: These warnings may cause trouble when distributing the package:
Warning: 'license: LGPL-2' is not a known version of that license. The known
versions are 2.1, 3. If this is not a mistake and you think it should be a
known version then please file a ticket.
```

Reply:
can't really follow what a known or unknown version is. Is it not known per this? https://github.com/haskell/cabal/blob/master/Cabal/src/Distribution/SPDX/LicenseId.hs#L237

Also, why would we mention `LGPL-3.0` when the user typed `LGPL-2`? We don't do that for `BSD2` either.